vendor.mod

```diff
@@ -121,6 +121,9 @@ require (
 	cloud.google.com/go v0.112.0 // indirect
 	cloud.google.com/go/longrunning v0.5.4 // indirect
 	github.com/AdamKorcz/go-118-fuzz-build v0.0.0-20231105174938-2b5cbb29f3e2 // indirect
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.16.0 // indirect
+	github.com/Azure/azure-sdk-for-go/sdk/internal v1.10.0 // indirect
+	github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v1.5.0 // indirect
 	github.com/agext/levenshtein v1.2.3 // indirect
```

Comment:
What's this dependency for?

Reply:
`github.com/agext/levenshtein` is for dockerfile hint suggestions on error related to https://github.com/moby/buildkit/pull/4925 but that's old.

Reply:
Sorry, meant the azure ones

Reply:
Ah the azure ones are related to the `gha` cache exporter. We need azblob dependency for v2 support: https://github.com/tonistiigi/go-actions-cache/pull/27/files#diff-0907ad0d6e3676122c4662a3486b1d8f378884c31e9488338fbc56e781168084R9-R10

Reply:
Right, but I wondered what changed here; didn't we support this before, or did we use some other SDK? I mean;

<img width="180" alt="Screenshot 2025-02-12 at 12 29 35" src="https://github.com/user-attachments/assets/c4755bf9-f956-4e97-8a05-ecde86286315" />

Reply:
Yeah it brings a lot :disappointed:

GitHub Cache service v2 relies on azblob API while v1 was common REST.